inabox/tests/integration_v2_test.go

```diff
@@ -135,9 +135,18 @@ var _ = Describe("Inabox v2 Integration", func() {
 			}
 		}
 
+		// These values are only used by the relay client to get chunks, which we do not call from the relay
```

Comment:
This looks just because of missing test coverage, then shall we add a call to test GetChunks?

Reply:
Talked to @ian-shim about this. He said the calls to `GetChunks()` are made from the operator nodes, not from this client instance. We are only using this client instance to call `GetBlobs()`.